database/migrations/2024_04_08_164938_create_artworks_tables.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up()
+    {
+        Schema::create('artworks', function (Blueprint $table) {
+            createDefaultTableFields($table);
+            $table->string('datahub_id');
+            $table->string('main_reference_number')->nullable();
+            $table->boolean('is_on_view')->nullable();
+            $table->string('credit_line')->nullable();
+            $table->string('copyright_notice')->nullable();
+            $table->decimal('latitude', 15, 13)->nullable();
+            $table->decimal('longitude', 15, 13)->nullable();
```

Comment:
I made the same mistake with longitude in the mobile app CMS. Unlike latitude, which ranges from -90 to 90 degrees, longitude ranges from -180 to 180, so it needs an additional significant digit in its precision.

For reference, see my migration to fix this in the mobile CMS repo:
https://github.com/art-institute-of-chicago/aic-mobile-cms/blob/develop/database/migrations/2023_12_05_001409_correct_longitude_columns.php